flywheel-io/core
# No root

### Changes
Site admin can do everything root mode does
  - slight exception for GET list (root=true or exhaustive=true needed to return entire list regardless of permissions)


### Test Plan
Integration tests
  - Tested that as_admin and as_root do the same thing (When calling get_all used the exhaustive flag)
  - Lowered the testing level to user with admin access on the group/project in order to test the auth layer (admin now skips it like root)

Manual tests
  - Can see all projects with root=true (can also use exhaustive=true)
  - Api calls returns single project without root=true (Frontend blocks it out)
  - Can upload to projects without root-true or permissions as long as user is site admin

This is not a breaking change to the frontend
### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md
